charts/incubator/tt-rss/docs/installation_notes.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Installation Notes
+---
+
+#### Credentials
```

Comment:
Whats the reason for `h4`?
Start with `h2` please (it should be the platform target 

Eg 

## TrueNAS SCALE

Reply:
Updated headings in https://github.com/truecharts/charts/pull/17293/commits/d865d669f209971e10cb315866d653ae7661bd50